src/core/include/openvino/runtime/allocator.hpp

```diff
@@ -61,22 +66,63 @@ class Tensor;
 /**
  * @brief Wraps allocator implementation to provide safe way to store allocater loaded from shared library
  *        And constructs default based on `new` `delete` c++ calls allocator if created without parameters
+ *        Accepts any [std::pmr::memory_resource](https://en.cppreference.com/w/cpp/memory/memory_resource) like
```

Comment:
is it OK that std::pmr::memory_resource is available since CXX17 https://en.cppreference.com/w/cpp/memory/memory_resource ?